koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -266,11 +268,11 @@ def bigquery_export_to_partition_mapping(self):
         except GoogleCloudError as err:
             err_msg = (
                 "Could not query table for partition date information."
-                f"\n  Provider: {self._provider_uuid}"
-                f"\n  Customer: {self.customer_name}"
-                f"\n  Response: {err.message}"
+                f"  Provider: {self._provider_uuid}"
+                f"  Customer: {self.customer_name}"
+                f"  Response: {err.message}"
             )
-            LOG.warning(err_msg)
+            LOG.warning(log_json(self.tracing_id, err_msg))
```

Comment:
It may be useful to add these fields to `self.context` for easy searching in Kibana. `self.context` already contains provider_uuid, provider_type, and account.
```suggestion
            extra_context = {"customer", self.customer_name, "response": err.message}
            LOG.warning(log_json(self.tracing_id, err_msg, self.context | extra_context))
```

Reply:
The union operator was added to `dict` in Python 3.9. Glad we're using newish Python. It's much nicer to read than dict unpacking, which was added in 3.5.
```python
LOG.warning(log_json(self.tracing_id, err_msg, {**self.context, **extra_context}))
```